es-ds-components/components/es-form-radio-cards.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+interface IOptions {
```

Comment:
Opted for the `I` in front so as to not mix-up  `p` and `P`, but [the vue docs](https://vuejs.org/guide/typescript/composition-api.html) seem to go with lower/uppercase for variable/interface so that's fine.

Change made via f02d866ef296a27d695bf435299cb83cc5593d17